api/app/services/ingestions/strategies/epub.rb

```diff
@@ -0,0 +1,100 @@
+module Ingestions
+  module Strategies
+    class Epub < Ingestions::Strategies::AbstractStrategy
+
+      def perform
+        compose Fetchers::URL, extension: "epub" if epub_url?
+        manifest
+      end
+
+      def determine_ingestibility
```

Comment:
With the change that comes from my comment above, this becomes simpler too, since it's already been fetched. You'll be able to lose the epub_url? method.